set.mm

```diff
@@ -91280,6 +91280,47 @@ without using the Axiom of Replacement or the Axiom of Power Sets (unlike
     UHABCDOUDUJLCUGJKZUHCUGUIPUDUKUHSUJCUGQRTUAUBUC $.
   $( $j usage 'f1imaenfi' avoids 'ax-pow' 'ax-rep'; $)
 
+  ${
+    $d D x $.  $d H x $.  $d A f g x $.  $d B f g x $.
+    sbthfilem.1 $e |- A e. _V $.
+    sbthfilem.2 $e |- D = { x |
+                   ( x C_ A /\ ( g " ( B \ ( f " x ) ) ) C_ ( A \ x ) ) } $.
+    sbthfilem.3 $e |- H = ( ( f |` U. D ) u. ( `' g |` ( A \ U. D ) ) ) $.
+    sbthfilem.4 $e |- B e. _V $.
+    $( Lemma for ~ sbthfi .  (Contributed by BTernaryTau, 4-Nov-2024.) $)
+    sbthfilem $p |- ( ( B e. Fin /\ A ~<_ B /\ B ~<_ A ) -> A ~~ B ) $=
```

Comment:
Since you use it only with setvars for A and B, you could simply state it with setvars, removing hypotheses 1 and 4.

More generally: looking at sbthfi: you use sbthfilem to prove the wanted statement with setvars and then use a vtocl* theorem to convert to classes. This is a common proof pattern, but now it may actually be easier to prove it using classes from the start thanks to the many recent additions of *g theorems, e.g., https://us.metamath.org/mpeuni/elpwg.html versus https://us.metamath.org/mpeuni/elpw.html or https://us.metamath.org/mpeuni/velpw.html

Reply:
As for the term, I think there is no standard. Maybe "closed form up to definitional hypotheses". See e.g., the comments of https://us.metamath.org/mpeuni/bj-pwcfsdom.html and https://us.metamath.org/mpeuni/bj-grur1.html.

I think one could actually do away with that lemma, by using the *g versions of theorems. If you can do that easily, then do it, else I think it's ok, and shouldn't block this MR. I'll let another person review (@jkingdon, since your worked on Schröder--Bernstein ?).

Reply:
I don't think there is an especially set way of describing "closed form minus the definitions" (I suspect I've used various wording), so I wouldn't worry about that much.

At least as far as I'm concerned, there isn't any particularly compelling need to change the form of this lemma. I think there's a preference for `A e. V` type existence hypotheses (in closed or deduction form, rather than inference form) but I don't see a need to treat that preference as blocking for this pull request